examples/gocounter/main.go

```diff
@@ -20,31 +21,46 @@ import (
 	"github.com/redhat-et/bpfd/clients/gobpfd"
 	"golang.org/x/sys/unix"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/credentials"
 )
 
 type Stats struct {
 	Packets uint64
+	Bytes   uint64
 }
 
+const (
+	DefaultRootCaPath     = "/etc/bpfd/certs/ca/ca.pem"
+	DefaultClientCertPath = "/etc/bpfd/certs/go-bpfd-client/go-bpfd-client.pem"
```

Comment:
This should be gocounter.pem.
The model we are after is one certificate per application.